src/Symfony/Component/DomCrawler/CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 CHANGELOG
 =========
 
+5.3.0
+-----
+
+ * Marked the `containsOption()`, `availableOptionValues()`, and `disableValidation()` methods of the
```

Comment:
maybe worth an UPGRADE* entry?

Reply:
not sure, it seems we didn't do that before when marking methods or classes as final

Reply:
we did https://github.com/symfony/symfony/search?l=Markdown&q=final :)

Reply:
@chalasr I only see matches in the changelog files :)